Portfolio/index.html

```diff
@@ -1,209 +1,236 @@
-            <div class="explore-btn">
-                <button>Explore</button>
-            </div>
-            <div class="project-text">
-
-
-
-                <img src="/screen2.png" alt="css img" width="400px" height="400px">
+            <div class="project-text1">
                 <h1><b>Taskbot, Real Time Task Manager</b></h1>
-                <p>React.js, typescript, Firebase</p>
-                <p><b>Developed a responsive e-commerce Website using React.js and Typescript.</b></p>
-                <p><b>Integrated Firebase for user authentication and firestore forproduct data storage.</b></p>
-                <p><b>Implemented secure payment processing with stripe API. </b></p>
-
+                <p>React.js, Firebase</p>
+                <p><b>Developed a dynamic task management application using React.js and Firebase.</b></p>
+                <p><b>Users can create, update, and delete tasks in time across devices. real-time across devices.</b>
+                </p>
+                <p><b>Integrated Firebase Authentication for secure user access.</b></p>
                 <div class="explore-btn">
                     <button>Explore</button>
                 </div>
+            </div>
+
+            <div class="project2">
 
-                <div class="project-text">
+                <img src="/screen2.png" alt="css img" width="400px" height="400px">
+                <div class="project-text2">
+                    <h1><b>Taskbot, Real Time Task Manager</b></h1>
+                    <p>React.js, typescript, Firebase</p>
+                    <p><b>Developed a responsive e-commerce Website using React.js and Typescript.</b></p>
+                    <p><b>Integrated Firebase for user authentication and firestore forproduct data storage.</b></p>
+                    <p><b>Implemented secure payment processing with stripe API. </b></p>
 
+                    <div class="explore-btn">
+                        <button>Explore</button>
+                    </div>
+                </div>
 
+                <div class="project3">
 
                     <img src="/screen3.png" alt="css img" width="400px" height="400px">
-                    <h1><b>FinanceSage, Personal Finance Tracker </b></h1>
-                    <p>Svelte.js, Firebase</p>
-                    <p><b>Created a lightweight personal finance tracking web app using Svelte.js and Firebase.</b></p>
-                    <p><b>Users can log income and expenses, visualize financial data and set budget goals Firebase
-                            Authentication for user management and Firestore for data storage.</b></p>
+                    <div class="project-text3">
+                        <h1><b>FinanceSage, Personal Finance Tracker </b></h1>
+                        <p>Svelte.js, Firebase</p>
+                        <p><b>Created a lightweight personal finance tracking web app using Svelte.js and Firebase.</b>
+                        </p>

```

Comment:
The commented-out 'margin-left' declaration in the .explore-btn button class should be removed for cleaner code.